lib/logidze/model.rb

```diff
@@ -38,11 +38,15 @@ def without_logging(&block)
     # Use this to convert Ruby time to milliseconds
     TIME_FACTOR = 1_000
 
+    attr_reader :logidze_requested_ts
+
     # Return a dirty copy of record at specified time
     # If time is less then the first version, then return nil.
     # If time is greater then the last version, then return self.
     def at(ts)
       ts = parse_time(ts)
+      @logidze_requested_ts = ts
```

Comment:
@palkan here's how I covered the case, when `at` doesn't change the object. Can we do better? I am not particularly happy with `attr_reader`